src/views/network/AntennaCriteriaCheck.vue

```diff
@@ -0,0 +1,347 @@
+                <b-tag type="is-danger" size="is-medium" v-if="props.row.antennaCriteria.fulfilmentReport === 'false'">No</b-tag>
+                <b-tag type="is-link" size="is-medium" v-if="props.row.antennaCriteria.fulfilmentReport === 'exception'">Exception</b-tag>
+              </template>
+            </b-table-column>
+
+            <b-table-column>
+              <b-button @click="openAntennaCriteriaInfo(props.row)" class="button is-link">
+                <span class="white"><font-awesome-icon :icon="['fa', 'eye']" /></span>
+              </b-button>
+            </b-table-column>
+
+            <b-table-column>
+              <b-button @click="openAntennaCriteriaModal(props.row)" class="button is-warning">
+                <span class="white"><font-awesome-icon :icon="['fa', 'pencil-alt']" /></span>
+              </b-button>
+            </b-table-column>
+
+          </template>
+
+          <template slot="empty">
+            <empty-table-stub />
+          </template>
+        </b-table>
+      </article>
+    </div>
+  </div>
+</template>
+
+<script>
+import { mapGetters } from 'vuex'
+import moment from 'moment'
+import AntennaCriteriaModal from './AntennaCriteriaModal.vue'
+import AntennaCriteriaInfo from './AntennaCriteriaInfo.vue'
+
+export default {
+  name: 'AntennaCriteriaCheck',
+  data () {
+    return {
+      bodies: [],
+      agorae: null,
+      selectedAgora: null,
+      showDetails: false,
+      hideSafeLocals: false,
+      isLoading: false
+    }
+  },
+  computed: {
+    ...mapGetters({
+      services: 'services',
+      loginUser: 'user'
+    }),
+    filteredBodies () {
+      if (!this.hideSafeLocals) return this.bodies
+      return this.bodies.filter(body => { return body.status === false })
+    }
+  },
+  methods: {
+    openAntennaCriteriaModal (row) {
+      this.$buefy.modal.open({
+        component: AntennaCriteriaModal,
+        hasModaLCard: true,
+        props: {
+          local: row,
+          agora: this.selectedAgora,
+          services: this.services,
+          showError: this.$root.showError,
+          showSuccess: this.$root.showSuccess,
+          router: this.$router
+        }
+      })
+    },
+    openAntennaCriteriaInfo (row) {
+      this.$buefy.modal.open({
+        component: AntennaCriteriaInfo,
+        hasModalCard: true,
+        props: {
+          local: row,
+          agora: this.selectedAgora,
+          services: this.services,
+          showError: this.$root.showError,
+          showSuccess: this.$root.showSuccess,
+          router: this.$router
+        }
+      })
+    },
+    toggleShowDetails () {
+      this.showDetails = !this.showDetails
+    },
+    toggleHideSafeLocals () {
+      this.hideSafeLocals = !this.hideSafeLocals
+    },
+    fetchAgorae () {
+      this.isLoading = true
+      this.axios.get(this.services['statutory'], { params: { type: 'agora' } }).then((response) => {
+        this.agorae = response.data.data
```

Comment:
Benefit of testing it on a relatively empty database is seeing how it works without any Agorae. Can you make it throw an error here if `this.agorae` is empty? Now it shows an error for 'Could not fetch bodies'